dev.dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM node:14.15.1-alpine
+
+# container filesystem config
+ADD . /app
+WORKDIR /app
+ENV HOST 0.0.0.0
+
+# reduce shell operations to a single layer
+RUN apk update && \
+    apk upgrade && \
+    apk add g++ make python3 && \ 
+    npm install
```

Comment:
So after fiddling around a bit, I was unable to use yarn to perform build and run operations. Still looking into it since yarn is pretty much using npm under the hood.